File: 7_Kyu_Kata/find_the_next_perfect_square/JavaScript/findTheNextPerfectSquare.js
// 7 kyu - Find the next perfect square!  [ ID: 56269eb78ad2e4ced1000013  (find-the-next-perfect-square) ]
// URL: https://www.codewars.com/kata/56269eb78ad2e4ced1000013
// Category: undefined  |  Tags: ALGEBRA | FUNDAMENTALS
// *****************************************************************************
function findNextSquare(sq) {
  if (Math.sqrt(sq) % 1 !== 0) {
    return -1
  } else {
    return (Math.sqrt(sq) + 1) * (Math.sqrt(sq) + 1)
  }
}

File: 7-kyu-kata/find-the-next-perfect-square/python/find_the_next_perfect_square.py
# 7 kyu - Find the next perfect square!  [ ID: 56269eb78ad2e4ced1000013  (find-the-next-perfect-square) ]
# URL: https://www.codewars.com/kata/56269eb78ad2e4ced1000013
# Category: REFERENCE  |  Tags: ALGEBRA | FUNDAMENTALS
# ******************************************************************************
def find_next_square(sq):
      return -1 if sq ** 0.5 % 1 else (sq ** 0.5 + 1) ** 2
